Answers.md

```diff
@@ -1,9 +1,14 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
+Semantic HTML is a system of applying HTML to a page that pays careful attention to the heirarchy of elements in such a way that the flow of the web page you're working on makes visual sense in addition to the flow of your code making heirarchical sense to any developer that were to read your code.
 
 2. Describe some differences between ```display: block;``` and ```display: inline;```.
+ Display: block creates a block element that will have space before and after it, and will take up all of the page width available to it. Display: inline creates an inline element which are often found nested inside of block-level elements. An element with display: inline will not have any additional white space before or after it by default, and the element will only take up the necessary space rather than taking up all available width like a block-level element.
 
 3. What are the 4 areas of the box model?
+content --> padding --> border --> margin 
 
 4. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
+You're using the cross axis, provided that your flex-direction setting remains default.
 
-5. What is the git command to commit staged changes as well as write a message? 
\ No newline at end of file
+5. What is the git command to commit staged changes as well as write a message? 
+git commit -m "Your commit message here"
```

Comment:
Great comprehension answers! Especially a deeper level of `display: inline` and `display: block`